doc/usage_cn.md

```diff
@@ -56,7 +56,7 @@ Use "paddlecloud flags" for a list of top-level flags
 
 不同的PaddlePaddleCloud集群环境会提供不同的分布式存储服务。目前PaddlePaddleCloud支持HDFS和CephFS。
```

Comment:
What's the relation of *usage_cn.md* and *tutorial.md*? *usage_cn.md* is for internal cluster and another one is for DLNEL?